Tests/SwiftDocCTests/LinkTargets/LinkDestinationSummaryTests.swift

```diff
@@ -162,7 +162,7 @@ class ExternalLinkableTests: XCTestCase {
             let summary = node.externallyLinkableElementSummaries(context: context, renderNode: renderNode)[0]
             
             XCTAssertEqual(summary.title, "MyClass")
-            XCTAssertEqual(summary.path, "/documentation/mykit/myclass")
+            XCTAssertEqual(summary.path, "/documentation/MyKit/MyClass")
```

Comment:
Is it possible the casing changes here will be an issue for any clients? This seems more correct to me either way.

Reply:
This won't be an issue if clients use the data format as intended but it could be an issue if a client tracked content over time by the entity's path instead of the entity's reference URL. 

The reference URL is meant to serve as the identifier of the entity (as well as the identifier of the page or on-page content that the entity represents). All other properties are intended as content.

However, in writing this and thinking about why we store both the `path` and the `referenceURL` I started to wonder if there are cases where a page's "path" (including the fragment, as noted in the documentation) would be different from the path and reference of its reference URL. 

[This comment in `PresentationURLGenerator.swift`](https://github.com/apple/swift-docc/blob/main/Sources/SwiftDocC/Model/Rendering/PresentationURLGenerator.swift#L55-L56) seems to indicate that that is possible. So perhaps computing the path from the reference URL will result in incorrect values in some situations.